src/components/SingleTrikotActual.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import {List, Image,Container} from "semantic-ui-react";
+import {List, Feed} from "semantic-ui-react";
 import { connect } from 'react-redux';
 import countries from "./countries";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'countries'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)